docs/dasharo-tools-suite/documentation.md

```diff
@@ -5,18 +5,18 @@
 Dasharo Tools Suite was prepared to run on x86 platforms, but we can confirm
 that it boots on the following platforms:
 
-* ASUS KGPE-D16
-* Dell OptiPlex 7010/9010
+* ASUS KGPE-D16,
+* Dell OptiPlex 7010/9010,
 * MSI PRO Z690-A DDR4 ([test
-  report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75))
+  report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75)),
 * NovaCustom NV4x ([test
-  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=536764189&range=A161))
+  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=536764189&range=A161)),
 * NovaCustom NS5x/7x ([test
-  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675&range=A174))
-* Protectli FW6/VP46xx
+  report](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=38447675&range=A174)),
+* Protectli FW6/VP46xx.
```

Comment:
MSI PRO Z690-A DDR5 is missing

Reply:
@Pokisiekk please provide a link then

Reply:
https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=777478871

Reply:
@Pokisiekk added https://github.com/Dasharo/docs/pull/369/commits/7ff1682f4c238da2ea588c82f0e2a8bb1545f783#diff-8cfee4c026a23b6eea51be5305d2c9db7d12139283fcec2ee3522ffffc0dc026R12